kmip/tests/unit/services/server/crypto/test_engine.py

```diff
@@ -1387,6 +1387,34 @@ def test_verify_signature_invalid_signing_algorithm(self):
              b'\xbd\x9b\x6f\x23\xc9\x39\xa7\xd4'
              b'\xf5\xbe\xb0\x9d\x92\xf0\x17\x56'
          )},
+        {'algorithm': enums.CryptographicAlgorithm.AES,
+         'cipher_mode': enums.BlockCipherMode.GCM,
+         'key': (
+             b'\x2c\xd6\xfd\x85\xf1\x30\x28\x38'
+             b'\x63\x53\xff\xa1\x52\x1d\x8d\x7b'
+             b'\xc8\xeb\xed\x26\xb1\x6d\x94\x40'
+             b'\x5f\x03\xf6\xda\x5d\xef\x2d\xa8'
+         ),
+         'iv_nonce': (
+             b'\xba\x7a\x97\x67\x0f\xbb\x02\x62'
+             b'\x24\x36\x92\x9d'
+         ),
+         'plain_text': (
+             b'\x8b\x4f\x7e\x75\x16\x31\xe7\x65'
+             b'\xdc\x13\xfa\x63\xf0\x2f\x63\x4b'
+         ),
+         'auth_additional_data': (
+             b'\x90\xee\x7e\x56\xf9\x59\x34\x76'
+             b'\x1c\x39\xab\x75\x37\x2a\xc2\xc6'
+         ),
+         'auth_tag_length': 8,
```

Comment:
After looking through the `pyca/cryptography` source, `auth_tag_length` is definitely supposed to be bytes (so 8 here is correct). Please update the above GCM test vector (L1376) and change the `auth_tag_length` there from 128 (bits) down to 16 (bytes).

Reply:
Good catch! Fixed.